app/src/main/res/values-land/strings.xml

```diff
@@ -0,0 +1,17 @@
+<resources>
+    <string name="app_name">Goose Eye</string>
+    <string name="button_back_text">Back</string>
+    <string name="pg_text">+</string>
+    <string name="name_avengers">Avengers:\nEnd Game</string>
+    <string name="tag_line_text">Action, Adventure, Fantasy</string>
+    <string name="review_text">REVIEWS</string>
+    <string name="story_line_text">Storyline</string>
+    <string name="story_line_details_text">After the devastating events of Avengers: Infinity War,
```

Comment:
If you delete this xml file nothing will change

Reply:
If you delete this xml file nothing will change.

It's a redundant duplicate. Take it easy, just remove it.